code/modules/events/radiation_storm.dm

```diff
@@ -67,4 +69,8 @@
 
 		sleep(600) // Want to give them time to get out of maintenance.
 
-		revoke_maint_all_access()
+		if(is_radstorm)
+			revoke_maint_all_access()
+			is_radstorm = 0
+		else

```

Comment:
This `else return` block is completely pointless and stops the function from exiting normally